### ISSUE TITLE
MetaQuery style connection tends to have higher loss

### ISSUE THREAD

**Issue by victkk** [DravenALG/VLANeXt#6]
Is this consistent with your experiment? Would you view this as evidence of KV cache style VLA is overfitting? (Since they does not show significant differences in real world rollout)

**Comment by victkk** [comment]
> Hi, from my experiments, I don't quite focus on the loss magnitude, since lower loss usually doesn't mean better performance, especially with diffusion-based losses. In diffusion loss in the LIBERO training, a loss between 0.04 and 0.05 is acceptable; more than 0.05 may be unstable training, and lower than 0.04 may be overfitting.
> 
> And I also think that the MetaQuery style connection may be a good choice to avoid overfitting. But in the current VLA, it needs overfitting to do the task well, especially in real-world evaluation, which needs a very low loss to achieve some performance.

I understand the loss magnitude does not mean anything, but could you provide an exact number for metaquery and kv cache style vla with diffusion loss? In my experiment the metaquery one's loss is about 3/2 of the kv cache one. I wonder what's this ratio in your experiment. (just hoping i didn't make mistakes in implementation)
